### DISCUSSION TITLE
FIL-RetroPGF-2 Project Showcase

### DISCUSSION THREAD

**Discussion by Onkar-san** [filecoin-project/community#713] [RetroPGF]
<h3>Introduction</h3>
This thread is the kick-off for the project showcase phase of the recently announced FIL-RetroPGF-2. 

The vision is to build on the success of FIL-RetroPGF round 1 and reward work that has measureably impacted the Filecoin ecosystem. RetroPGF is an experimental mechanism to realize this goal of “impact = profit.” 

Round 2 involves distributing 300K FIL to projects that have shown impact in the previous 6 months (April - September 2024).

<br>
<h3>Overview of Timelines</h3>

- **Project Showcase: Sept 30 - Oct 14**
- Application Intake: Oct 14 - Nov 4
- Badgeholder Voting: Nov 11 - Nov 25
- Vote Collation and Funding Distribution: Dec. onwards

<br>
<h3>What is the Showcase?</h3>

In Round 1, applications were pre-screened for eligibility in a Nomination Phase before being invited to apply. This time, eligibility will be checked directly in the Application stage. However, applicants will still have a chance to show off their projects ahead of the full application in a visibility stage called the Showcase phase. 

<p>This is intended to give an additional platform for projects to share and promote what they’ve been building, and to keep the excitement and community engagement of the Nomination phase.

Anyone may showcase their project for FIL-RetroPGF-2! Specific instructions for how to showcase are listed below.

<br>
<h3>How to Participate?</h3>

1. **Submit a GitHub comment** – Briefly describe your project below with the following details:

    - **Project Name**
    - **50-word description of your project**
    - **Website URL/github/information link**
     

2. **Spread the word** – Share your submission on Twitter or other social media, tagging the Filecoin and FIL-RetroPGF account to engage the community and maximize visibility. A example template post is given below:

> We’re taking part in @Filecoin’s FIL-RetroPGF Round 2 with @FILRetroPGF! Check out our project here: [Link]!
#filretropgf #filretropgf2 #filecoin

**Important Note:** _This round will implement a x1.25 multiplier for spreading the word. Badgeholders will be asked to increase the score of projects who showcase their work and promote the FIL-RetroPGF round by a factor of x1.25. This is because increasing the visibility of public good funding and your impact on the ecosystem is valuable to FIL-RetroPGF, and will thus positively impact your application._ 

**_This tweet counts as a promotion!_**

<br>
<h3>Project Categories for Round 2</h3>
Filecoin-RetroPGF-2 will distribute 300K FIL to public goods that support Filecoin in one of the following categories:

- Infrastructure + Dependencies: software used to build or deploy the FIL Stack; contributions to protocols or standards upon which the FIL Stack runs; experiments that support future development of the core FIL Stack protocol.
- Tooling + Utilities: work that helps builders create applications on Filecoin mainnet, build on the FIL Stack, or use applications built on Filecoin.
- Education & Outreach: work to spread awareness and knowledge of how Filecoin works. Both technical and social contributions are welcome!
- Protocol Research & Development: work to improve the Filecoin protocol, to include both theoretical contributions as well as implementations and prototypes are in scope.
- Collective Governance: Work that impacted Filecoin's governance participants or helped bring new governance participants into the network.
- Products & End User UX: Work that helped improve the end-user experience and adoption of Filecoin technology.
_**Note:** The Filecoin Stack refers to the OSS code powering Filecoin, its dependencies, and related codebases._

The purpose of this thread is to provide visibility and feedback on ecosystem projects, and to showcase the impact being created. 

**Project Showcases will close by 14th October 2024.**

**Comment by wolfduan** [comment]
github
